src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private String line;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
the code is good, but lets try to divide it a bit
Use this pattern:

String data = readFromFile(fileName);
calculateStatisitc(data);
writeStatisticToFile(fileName);